rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/internal_interpreter.kt

```diff
@@ -61,7 +61,8 @@ class InterpreterStatus(
     val indicators: HashMap<IndicatorKey, BooleanValue>,
     var returnValue: Value? = null,
     var params: Int = 0,
-    var inzsrExecuted: Boolean = false
+    var inzsrExecuted: Boolean = false,
+    var dbFileMap: DBFileMap
```

Comment:
if this property is immutable, and I think yes, you should use the keyword `val`

Reply:
Done